### PR TITLE
Nft current balance

### PR DESCRIPTION
**Pull request by Shan-barcha**

### CODE REVIEW

**Comment by hunzai** [comment]
@Shan-barcha can you fix the conflicts? Also check why deployment is failing

**Comment by hunzai** [comment]
It fails because of lint

```

./pages/index.tsx
--
10:05:20.395 | 4:8  Error: 'Card' is defined but never used.  no-unused-vars
10:05:20.395 | 5:8  Error: 'image' is defined but never used.  no-unused-vars
```

make sure you run lint before pushing code

**Comment by hunzai** [comment]
run 
`npm run lint` to findout what is wrong with code and fix it

**Comment by hunzai** [comment]
@Shan-barcha please use a standard to name the branch. Either use camel case or lower case e.g.
nft-current-branch or NftCurrentBranch. It is one of coding standards

File: components/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Image from "next/image";
+
+const Card = ({ item }) => {
+  return (
+    <div className="flex justify-center mt-5">
+      <div className="rounded-lg shadow-xl bg-white max-w-[300px]  lg:max-w-xs xl:max-w-sm ">
+        <a href="#!">
+          <Image
```
Comment (hunzai) [inline components/Card.tsx:9]: Use responsive design approach, see https://tailwindcss.com/docs/responsive-design

Comment (hunzai) [reply components/Card.tsx:9]: Test on small and bigger browser screen sizes

File: components/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Image from "next/image";
+
+const Card = ({ item }) => {
+  return (
+    <div className="flex justify-center mt-5">
```
Comment (hunzai) [inline components/Card.tsx]: use width to make the div responsive. See https://tailwindcss.com/docs/max-width

**Comment by Shan-barcha** [comment]
i added coverage folder in git ignore ,since it should not be on git hub . this folder auto generate when we run test.

File: components/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+          <div className=" text-gray-900 text-base mb-4 break-words  w-full">
+            {item.policyId}
+          </div>
+          <div className="flex justify-between items-center">
+            <h4 className="font-bold text-center lg:text-lg text-blue-800 ">
+              200$
+            </h4>
+            <button
+              type="button"
+              className=" inline-block px-6 py-2.5 bg-blue-600 text-white font-medium text-xs leading-tight uppercase rounded shadow-md hover:bg-blue-700 hover:shadow-lg focus:bg-blue-700 focus:shadow-lg focus:outline-none focus:ring-0 active:bg-blue-800 active:shadow-lg transition duration-150 ease-in-out text-center lg:text-[16px] "
```
Comment (hunzai) [inline components/Card.tsx]: always try to make it as simple as possible. In this case use `rounded-full` class see https://tailwindcss.com/docs/border-radius#pill-buttons

**Comment by Shan-barcha** [comment]
sir is that not responsive, I made it responsive.

File: components/Card.tsx (hunk 1 above)
Comment (hunzai) [reply components/Card.tsx:9]: can you deploy and check on bigger screens. Read docs on how to make it responsive. I can't see first and last card properly on my screen. 

image

**Comment by Shan-barcha** [comment]
sir, why is responsiveness not working on your browser?  in my browser it is responsive.

**Comment by Shan-barcha** [comment]
@hunzai  sir please review it !

**Pull request merged**